xknx.yaml

```diff
@@ -1,8 +1,11 @@
 
 general:
     own_address: '15.15.249'
-    rate_limit: 20
+    rate_limit: 18
```

Comment:
Why did you change this? Testing?

Reply:
right. just for the tests to have a different value than default. 🤔maybe we should rename this file to something like `test.yaml` or `example.yaml`? Or just write some comments to point that out.

Reply:
Yea maybe, From what I know it was just an example so maybe example.yml might do the trick. Anyway, as it's unrelated to the contents of this PR i will resolve this conversation